<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>819</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>DATA</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/icon/proc4.png</normaloff>:/icon/proc4.png</iconset>
  </property>
  <property name="accessibleDescription">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
	
	border-color: #000;
	border-width:2px;
	border-radius:10px;
	border-style:solid;
}
QProgressBar{
	
	background-color: rgb(255, 129, 39);
	font-size:15px;
	
	color: rgb(255, 255, 255);
	selection-background-color:rgb(11, 161, 0)
}

QLabel{
	color: rgb(255, 0, 0);
}
QPushButton{
	border-radius:10px;
	border-style:solid;
	border-width:2px;
	border-color:red;
	background-color:white;
}</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="CustomeEDT" name="txtRam">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>204</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCode">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtData">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>480</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtStack">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblRam">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>209</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RAM Capacity </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>486</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Code Seg</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCpu">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>206</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cpu</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCore">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>345</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Core Count </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStack">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>346</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stack Seg</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCpu">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>202</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCore">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblData">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>486</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Data Seg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>610</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>610</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-11</y>
      <width>661</width>
      <height>181</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/img.jpg);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>615</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/attantion.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>610</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
image: url(:/icon/stack.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="icnStack" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>380</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/attantion.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="icnRam" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/attantion.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="icnData" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>520</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/attantion.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="icnCore" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/attantion.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="icnCpu" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/attantion.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="icnCode" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/attantion.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="errCore">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>384</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter a Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="errRam">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>243</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter a Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="errCode">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>523</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter a Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="errStack">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>383</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter a Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="errCpu">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>243</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter a Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="errData">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>525</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter a Number</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_7" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>740</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color:red;
font-size:20px;</string>
     </property>
     <property name="text">
      <string>&gt;Data Form</string>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>800</y>
      <width>331</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <zorder>progressBar</zorder>
   <zorder>txtCpu</zorder>
   <zorder>txtRam</zorder>
   <zorder>txtCode</zorder>
   <zorder>txtData</zorder>
   <zorder>txtStack</zorder>
   <zorder>lblRam</zorder>
   <zorder>lblCpu</zorder>
   <zorder>lblStack</zorder>
   <zorder>txtCore</zorder>
   <zorder>lblCore</zorder>
   <zorder>lblCode</zorder>
   <zorder>lblData</zorder>
   <zorder>pushButton</zorder>
   <zorder>Exit</zorder>
   <zorder>widget</zorder>
   <zorder>widget_2</zorder>
   <zorder>widget_3</zorder>
   <zorder>icnStack</zorder>
   <zorder>icnRam</zorder>
   <zorder>icnData</zorder>
   <zorder>icnCore</zorder>
   <zorder>icnCpu</zorder>
   <zorder>icnCode</zorder>
   <zorder>errCore</zorder>
   <zorder>errRam</zorder>
   <zorder>errCode</zorder>
   <zorder>errStack</zorder>
   <zorder>errCpu</zorder>
   <zorder>errData</zorder>
   <zorder>widget_7</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionRam"/>
    <addaction name="actionCpu"/>
    <addaction name="actionCode_seg"/>
    <addaction name="actionStack_seg"/>
    <addaction name="actionData_seg"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>about</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionVersion"/>
    <addaction name="actionAbout"/>
    <addaction name="actionDeveloper"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>new</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionRam">
   <property name="text">
    <string>ram</string>
   </property>
  </action>
  <action name="actionCpu">
   <property name="text">
    <string>cpu</string>
   </property>
  </action>
  <action name="actionCode_seg">
   <property name="text">
    <string>code seg</string>
   </property>
  </action>
  <action name="actionStack_seg">
   <property name="text">
    <string>stack seg</string>
   </property>
  </action>
  <action name="actionData_seg">
   <property name="text">
    <string>data seg</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDeveloper">
   <property name="text">
    <string>Developer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomeEDT</class>
   <extends>QLineEdit</extends>
   <header>customeedt.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
